src/App.js

```diff
@@ -1,25 +1,16 @@
-import logo from './logo.svg';
 import './App.css';
+import React, { Component } from "react";
+import FetchComponent from "./components/FetchComponent";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+
+  render() {
+    return (
+      <div>
+        <FetchComponent title="Fetch API" />
```

Comment:
Improve Component naming, the component will be representing the table, not the fetch process.

Reply:
The `FetchComponent` component has been renamed to [`UserTable.js`](https://github.com/jql6/mentorsNetwork-FrontEnd/blob/FetchAPI/src/components/UserTable.js) to better represent its purpose.